Undefined.ts

```diff
@@ -0,0 +1,5 @@
+export namespace Undefined {
+	export function not<T>(value: T | undefined): value is T {
+		return value != undefined
```

Comment:
i think this is a case of `!==` as `!=` will misbehave for stuff like `null`
![image](https://github.com/user-attachments/assets/c807fc67-908e-4788-b761-93c347c0782f)